assets/js/app.js

```diff
@@ -21,5 +21,6 @@ import "phoenix_html"
 
 // import socket from "./socket"
 
-import "./header"
-import "./filter_players"
+import "./header";
+import "./filter_players";
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)